tests/python/unittest/test_operator.py

```diff
@@ -2840,6 +2840,13 @@ def test_transpose():
             assert_allclose(np.transpose(x.asnumpy()), y.asnumpy())
 
 
+@with_seed()
+def test_larger_transpose():
```

Comment:
Why does this say larger_transpose ? is it a bug with specific sizes ?

Reply:
@ChaiBapchya I think we should merge the test with the current tests of transpose.

Reply:
Current test_transpose has size less than the block_size
This PR has blocksize 32*32 (and edge case didn't get tested with existing set of tests)
Hence added this test to catch it.

Reply:
can we add this with the current tests of transpose as @sxjscience suggests. otherwise LGTM